File: ReleaseNotes.API/ReleaseNotes.API/Services/Project/Models/ReleaseNotes/ReleaseNotesResponseModel.cs
namespace ReleaseNotes.API.Services.Project.Models.ReleaseNotes;

public class ReleaseNotesResponseModel
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string Name { get; set; } = string.Empty;
    public string LatestVersion { get; set; } = string.Empty;
    public DateTimeOffset CreatedOnUtc { get; set; }
    public string CreatedBy { get; set; } = string.Empty;

    public IEnumerable<ReleaseNoteReleaseResponseModel> Releases { get; set; } = [];
}

File: ReleaseNotes.UI/src/services/project/models/release-notes/release-notes-response-model.ts
import { ReleaseNoteReleaseResponseModel } from "./release-note-release-response-model";

export interface ReleaseNotesResponseModel {
  id: string;
  name: string;
  latestVersion: string;
  createdOnUtc: Date;
  createdBy: string;
  releases: ReleaseNoteReleaseResponseModel[];
}
